components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -643,7 +643,7 @@ internal sealed class SubscriptionWrapper : ISubscription {
                     }
 
                     if (_currentlyMonitored == null) {
-                        _logger.Information("DataChange for subscription: {Subscription} having no monitored items yet",
+                        _logger.Verbose("DataChange for subscription: {Subscription} having no monitored items yet",
```

Comment:
>Verbose [](start = 32, length = 7)

this should not happen normally, therefore we need to know it happens. Again the logic was changed and this should never appear

Reply:
Will revert, only 14 hits